plugins/scaffolder-backend-module-confluence-to-markdown/src/actions/confluence/confluenceToMarkdown.ts

```diff
@@ -66,7 +66,6 @@ export const createConfluenceToMarkdownAction = (options: {
               'Paste your Confluence url. Ensure it follows this format: https://{confluence+base+url}/display/{spacekey}/{page+title} or https://{confluence+base+url}/spaces/{spacekey}/pages/1234567/{page+title} for Confluence Cloud',
             items: {
               type: 'string',
-              default: 'Confluence URL',
```

Comment:
This is actually not part of the `Schema` type, wondering if it should be and that the type is wrong.

Should the type actually be `JSONSchema7` instead from `json-schema`, thinking that this could break actions otherwise. It looks like the types were not inferred properly before, and now they are. @freben